cob_bringup/drivers/laser_lms1xx.launch

```diff
@@ -3,18 +3,21 @@
 
 	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
-	<arg name="name" default="front"/>
+	<arg name="name"/>
+
+	<!-- load parameter -->
+	<rosparam ns="laser_$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/laser_$(arg name).yaml"/>
 
 	<!-- start laser driver -->
-	<node name="laser_$(arg name)" pkg="cob_sick_lms1xx" type="lms100" respawn="false" output="screen" >
-		<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/laser_$(arg name).yaml"/>
-		<remap from="laser_$(arg name)/scan" to="/scan_$(arg name)_raw"/>
+	<node ns="laser_$(arg name)" pkg="cob_sick_lms1xx" type="lms100" name="laser_$(arg name)" respawn="false" output="screen" >
+		<remap from="scan" to="/scan_$(arg name)_raw"/>
 	</node>
 
 	<!-- laser filter -->
-	<node name="laser_$(arg name)_filter" pkg="cob_sick_s300" type="cob_scan_filter" respawn="false" output="screen" ns="laser_front">
-		<remap from="/laser_$(arg name)/scan" to="/scan_$(arg name)_raw"/>
-		<remap from="/laser_$(arg name)/scan_filtered" to="/scan_$(arg name)"/>
-	</node>
+	<include file="$(find cob_bringup)/drivers/laser_scan_filter.launch">
+		<arg name="robot" value="$(arg robot)"/>
+		<arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)"/>
+		<arg name="name" value="$(arg name)"/>
+	</include>
```

Comment:
scan filtering is  now outsourced into separate launch file, which is also used for simulation!